source/enso_doc/enso.rst

```diff
@@ -0,0 +1,81 @@
+.. _enso:
+
+--------------------
+Introduction to ENSO
+--------------------
+
+.. contents::
+
+What is ENSO ?
+==============
+
+``ENSO`` is a utility program to create and refine Conformer, Rotamer-Ensembles and finally prepare all files necessary to generate NMR spectra using the ``ANMR`` program. ``ENSO`` and additional necessary resources can be found at [latest release page](https://github.com/grimme-lab/enso/releases/latest) and [latest release page](https://github.com/grimme-lab/cefine/releases/latest).  
+
+The Workflow to get the final spectrum is detailed in the following graphic.
+
+.. figure:: ../../figures/enso/enso_get_spectrum.png
+	:scale: 25%
+	:align: center
+	:alt: Workflow to get a calculated NMR spectrum
+
+	NMR spectrum calculation workflow.
+
+
+
+Short summary
+=============
+
+The ``ENSO`` program is designed to rerank the Conformer Rotamer Ensemble (CRE) generated by ``CREST`` (Conformer Rotamer Ensemble Search Tool) at DFT level and perform subsequent NMR-property calculations. ``ENSO`` interfaces to different QM-CODES (TURBOMOLE, ORCA, xTB) and processes the data. The program is subdivided into four parts. 
```

Comment:
Same here, are you talking about the program `crest`? Also, the xTB method or the `xtb` program?